pkg/kubelet/dockershim/libdocker/testing/mock_client.go

```diff
@@ -0,0 +1,407 @@
+// +build !dockerless
+
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Generated code, generated via: `mockgen -source pkg/kubelet/dockershim/libdocker/client.go  -destination pkg/kubelet/dockershim/libdocker/testing/mock_client.go -package testing`
```

Comment:
thanks for documenting this. may be worth logging an issue so we won't forget to do this once this PR lands

Reply:
agree with @dims